templates/partial/_careers-available-roles.html

```diff
@@ -1,3 +1,7 @@
+<script>
+  const vacancies = {{ vacancies_json | tojson }};
+</script>
```

Comment:
`/careers/engineering` is not working, because for that page you don't have `vacancies_json`. So you could solve this by wrapping this script in a if statement
```suggestion
{% if vacancies_json %}
<script>
 const vacancies = {{ vacancies_json | tojson }};
</script>
{% endif %}
```

Reply:
Now check if `/careers/engineering` works